core/src/main/java/org/apache/calcite/plan/volcano/RuleQueue.java

```diff
@@ -265,7 +264,7 @@ private void checkDuplicateSubsets(Deque<RelSubset> subsets,
    * for a particular
    * {@link VolcanoPlannerPhase phase of the planner's execution}.
    */
-  private static class PhaseMatchList {
+  static class PhaseMatchList {
     /**
```

Comment:
Why change it ?

Reply:
This class is referenced in a test case.

Reply:
I don't think it is a good practice to modify it just for the test case. Can we change the test case?

Reply:
Sure. Sounds reasonable. I have revised the test case, and restored the modifier.